include/ginkgo/core/reorder/rcm.hpp

```diff
@@ -224,6 +160,71 @@ class Rcm : public EnablePolymorphicObject<Rcm<ValueType, IndexType>,
 
 
 }  // namespace reorder
+
+
+namespace experimental {
+namespace reorder {
+
+
+using rcm_starting_strategy = gko::reorder::starting_strategy;
+
+
+/**
+ * @copydoc gko::reorder::Rcm
+ */
+template <typename IndexType = int32>
+class Rcm : public EnablePolymorphicObject<Rcm<IndexType>, LinOpFactory>,
```

Comment:
I guess this class should replace the current one, right? Then the current one should be deprecated.

Reply:
If you don't want to replace the old one, then perhaps this would fit better in the benchmark directory.

Reply:
Yes, it is intended for deprecation, but I wanted to avoid having too many tests that require deprecation warnings turned off.

Reply:
Ok, just wanted to know the future direction of this. But in that case we will not get around deprecating the old one. I would also favor @pratikvn suggestion of moving all these changes into a separate PR.